pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
+
+| All Keywords                | Specification | Draft introduction | Removed | Changed                                                                                      |
+| --------------------------- | ------------- | ------------------ | ------- | -------------------------------------------------------------------------------------------- |
+| `$anchor`                   | Core          | 2019-09            | No      | Partially replaced `$id`                                                                     |
+| `$comment`                  | Core          | 07                 | No      |                                                                                              |
+| `$id`                       | Core          | 06                 | No      | Replaced `id`                                                                                |
+| `$defs`                     | Core          | 2019-09            | No      | Replaced `definitions`                                                                       |
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
+| `$recursiveAnchor`          | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`                                                                 |
+| `$recursiveRef`             | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`                                                                    |
+| `$ref` (legacy)             | Core          | 03                 | 2019-09 | Replaced by `$ref`                                                                           |
+| `$ref`                      | Core          | 2019-09            | No      | Replaced `$ref` (legacy)                                                                     |
+| `$schema`                   | Core          | 03                 | No      | No                                                                                           |
+| `$vocabulary`               | Core          | 2019-09            | No      | No                                                                                           |
+| `additionalItems`           | Core          | 03                 | 2020-12 | Replaced by `items`                                                                          |
```

Comment:
In 2019-09, this keyword was modified to produce annotations

Reply:
I don't think we need to note that things started producing annotations in 2019-09. Nothing produced annotations before 2019-09. I think it's given or at least can be generally stated once at the top of the table that annotations only became a thing in 2019-09.